includes/definitions/aprisa.yaml

```diff
@@ -0,0 +1,14 @@
+os: aprisa
+text: 'Aprisa XE'
```

Comment:
If this is to deal with the Aprisa range then please remove the XE. If it's specific to the XE then the os name needs to be updated to reflect that.